client/state/themes/selectors/is-live-preview-supported.js

```diff
@@ -0,0 +1,139 @@
+import config from '@automattic/calypso-config';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { isSimpleSite } from 'calypso/state/sites/selectors';
+import {
+	isWporgTheme,
+	isThemeActive,
+	isFullSiteEditingTheme,
+	getTheme,
+	isExternallyManagedTheme,
+	canUseTheme,
+} from '../selectors';
+
+/**
+ * Hardcoded list of themes that are not compatible with Block Theme Previews.
+ * This list should be removed once they are retired.
+ *
+ * @see pekYwv-284-p2
+ */
+const NOT_COMPATIBLE_THEMES = [
+	// premium themes (premium/*)
+	'alonso',
+	'baxter',
+	'byrne',
+	'kerr',
+	'munchies',
+	'payton',
+	'skivers',
+	'thriving-artist',
+
+	// free themes (pub/*)
+	'ames',
+	'antonia',
+	'appleton',
+	'arbutus',
+	'attar',
+	'barnett',
+	'bennett',
+	'calvin',
+	'calyx',
+	'course',
+	'dorna',
+	'farrow',
+	'geologist-blue',
+	'geologist-cream',
+	'geologist-slate',
+	'geologist-yellow',
+	'hari',
+	'heiwa',
+	'jackson',
+	'kingsley',
+	'marl',
+	'meraki',
+	'quadrat-black',
+	'quadrat-green',
+	'quadrat-red',
+	'quadrat-white',
+	'quadrat-yellow',
+	'quadrat',
+	'russell',
+	'seedlet-blocks',
+	'winkel',
+];
+
+const isNotCompatibleThemes = ( themeId ) => {
+	return NOT_COMPATIBLE_THEMES.includes( themeId );
+};
+
+/**
+ * Check if Live Preview is supported.
+ *
+ * The scenarios where the Live Preview does NOT support;
+ * - On both Simple and Atomic sites;
```

Comment:
We also need to add a check whether the user is logged in or not. Since the BTP doesn't work while logged out.